src/features/island/fisherman/FishingChallenge.tsx

```diff
@@ -127,7 +127,7 @@ export const FishingChallenge: React.FC<Props> = ({
     from: { transform: `rotate(${castFrom}deg)` },
     to: { transform: `rotate(${castTo}deg)` },
     onRest: () => {
-      if (misses >= 2) {
+      if (misses >= 2 && hits < health) {
```

Comment:
Small bug fix where `onRest` was triggering a failure, after a winning game won on 2/3 misses.

Reply:
OMG! 😱 

That got me so many times 😆